### PR TITLE
Plugin unit tests

### PR DESCRIPTION
**Pull request by mqueme**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.

### CODE REVIEW

File: app/bundles/PluginBundle/Helper/IntegrationHelper.php [hunk 1]
```diff
@@ -240,21 +250,24 @@ public function getIntegrationObjects($specificIntegrations = null, $withFeature
             }
 
             if (!isset($integrations[$integrationName])) {
-                $integration     = $available[$integrationName];
```
Comment (dongilbert) [inline app/bundles/PluginBundle/Helper/IntegrationHelper.php]: I need to add back the manual class name building & instantiation to keep BC for 3rd party plugins, at least until 3.0. Here, line 105 and line 166

File: app/bundles/NotificationBundle/Config/config.php [hunk 2]
```diff
@@ -146,6 +146,14 @@
                 ],
             ],
         ],
+        'integrations' => [
```
Comment (dongilbert) [inline app/bundles/NotificationBundle/Config/config.php]: @mqueme Here is an example of how to register the integrations in your bundle config.

Comment (mqueme) [inline app/bundles/NotificationBundle/Config/config.php]: tnx

**Comment by escopecz** [comment]
I'm getting `500 Internal Server Error - PHP Notice - Undefined index: GithubAuth at app/bundles/PluginBundle/Helper/IntegrationHelper.php:299` when I try to access the plugins. How come it works for you and not for me?

**Comment by escopecz** [comment]
This error is caused by @alanhartless's https://github.com/alanhartless/mautic-sso-plugin. This raises a the question whether the problem is in Alan's plugin or if this PR breaks BC.

**Comment by virlatinus** [comment]
The tests are passing! Thanks! +1

**Pull request merged**